bigquery/query.js

```diff
@@ -0,0 +1,176 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START complete]
+/**
+ * Command-line application to perform an synchronous query in BigQuery.
+ *
+ * This sample is used on this page:
+ *
+ *   https://cloud.google.com/bigquery/querying-data#syncqueries
+ */
+
+'use strict';
+
+// [START auth]
+// By default, gcloud will authenticate using the service account file specified
+// by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use the
+// project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/guides/authentication
+var BigQuery = require('@google-cloud/bigquery');
+
+// Instantiate the bigquery client
+var bigquery = BigQuery();
+// [END auth]
+
+// [START sync_query]
+/**
+ * Run a synchronous query.
+ * @param {string} query The BigQuery query to run, as a string
+ * @param {function} callback Callback function to receive query results.
+ */
+function syncQuery (query, callback) {
+  if (!query || typeof query !== 'string') {
+    return callback(Error('"query" is required, and must be a string!'));
```

Comment:
I'm assuming I should keep the `callback` portion, e.g. `return callback(new Error(...))`?